eureka-client/src/main/java/com/netflix/discovery/DiscoveryClient.java

```diff
@@ -856,9 +856,9 @@ public Applications getApplications(String serviceUrl) {
                 logger.debug(PREFIX + "{} -  refresh status: {}", appPathIdentifier, response.getStatusCode());
                 return response.getEntity();
             }
-            logger.error(PREFIX + "{} - was unable to refresh its cache! status = {}", appPathIdentifier, response.getStatusCode());
+            logger.info(PREFIX + "{} - was unable to refresh its cache! status = {}", appPathIdentifier, response.getStatusCode());
```

Comment:
I'd also improve this log msg, e.g. it'd be useful if it stated that this is a periodic background process and that it will be retried in X seconds. Additionally, it may be valuable to preserve the exception details in some form.

Reply:
@troshko111 Can you elaborate on preserve the exception details? I enquired about capturing the stack track, but we agreed that we wouldn't.  The log already captures the exception's detail message string (oddly labelled as "status=", but it is there).  

Good idea on the refresh interval; I've added that to my local branch and will push once I've got clarification on the above.

Reply:
I'm inclined to believe I was wrong, I'm trying to find some sample history to have something to look at, but iirc the actual reason would be couple frames down so printing the msg only may be insufficient and having more stack info would be best.